integrations/hello-world-ng11-ivy/cypress/integration-ssr/issue-1646-wrong-ssr-state.spec.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+describe('Select decorator returning state from the wrong store during SSR (https://github.com/ngxs/store/issues/1646)', () => {
```

Comment:
can this code be added as another PR?

Reply:
or it's imposible?

Reply:
This is one of the issues solved by this fix. I think that this is appropriate to be here.